third_party/check.BUILD

```diff
@@ -22,6 +22,10 @@ cmake(
         "CMAKE_INSTALL_LIBDIR": "lib",
         "HAVE_SUBUNIT": "0",
     },
+    generate_args = [
+        "-GNinja",
```

Comment:
Wouldn't this require all macOS devs to have Ninja preinstalled on their system?

Reply:
Let me double check.

I assumed that since they go through the trouble of bootstrapping `gnu make` that they also provide a ninja binary or something.

But to be fair I didn't notice if one was getting downloaded/built.

Reply:
I checked each platform and the build scripts for the cmake builds all reference  ninja instances that live in the `execroot`

Looks like pre-built instance on x86_64-darwin and x86_64-linux,
and built from source on aarch64-darwin.

Reply:
👍